contrib/inventory/ec2.py

```diff
@@ -721,6 +654,51 @@ def get_elasticache_replication_groups_by_region(self, region):
         for replication_group in replication_groups:
             self.add_elasticache_replication_group(replication_group, region)
 
+    def get_autoscaling_instances_by_region(self, region):
+        ''' Makes an AWS API call to the list of Autoscale instances
+         in a particular region with tags of their Autoscale group
+         if the "propagate_at_launch" is set '''
+
+        try:
+            conn_autoscale = self.connect_to_aws(autoscale, region)
+            conn_ec2 = self.connect_to_aws(ec2, region)
+            if conn_autoscale and conn_ec2:
+                reservations = []
+                autoscale_groups = conn_autoscale.get_all_groups()
+                for autoscale_group in autoscale_groups:
+                  if autoscale_group.name:
+                    tags = {}
+                    for tag in autoscale_group.tags:
+                      if tag.propagate_at_launch:
```

Comment:
This code mixes 2 and 4-space tabs, which isn't in line with the style guide (pretty much PEP8) that we use.